src/common/views.py

```diff
@@ -32,12 +32,18 @@
 from shared.handle_get import *
 import os
 import shutil
+import requests
+import semver
 
 
 def common_list_view(request):
     context = dict()
-    context['release_version'] = dict()
     template = 'common/common_list.html'
+    context['current_app_version'] = current_app_version()
+    context['curr_module_id'] = 'id_internals_module'
+    return render(request, template, context)
+
+def current_app_version():
```

Comment:
pls move this to common.helper.py instead of views.py
Also name it get_current_app_version.  you have defined variable with same name below

Reply:
Changes made